### PR TITLE
Enable lock/unlock

### PR DESCRIPTION
**Pull request by danicaerediano**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -90,4 +90,5 @@ dist/
 .vscode/
 temp/
 .eslintcache
-.env.local
\ No newline at end of file
+.env.local
+.env
```
Comment (pmigueld) [inline .gitignore:94]: This shouldn't be ignored (if you need to customize the environment use `.env.local`). This is the template for environment variables.

**Review by pmigueld — CHANGES_REQUESTED** [review]

**Comment by pmigueld** [comment]
Tested and working in Android 10 but nothing happens when pressing the Lock button in iOS.

File: .gitignore (hunk 1 above)
Comment (danicaerediano) [reply .gitignore]: @pmigueld updated

**Review by pmigueld — APPROVED** [review]

**Comment by jannahadlaon** [comment]
**Tested on:** 
MOCK_io.mosip.residentapp-0.4.0-rc3-17-5f7c994-develop-temp-dirty-newlogic_20221003_0914.apk - **Android**
0.4.0 (9.2) - **iOS**

**Resident app build:** 
v0.4.0 R3-17-5f7c994 (android)
v0.4.0 (9.2) -iOS

**MOSIP server**: Mock

Device:
 Samsung Galaxy A23
 iPhone 11
 
`Lock` button is not working when clicked. So as the `Rename` button

**Assign back to dev** 

https://user-images.githubusercontent.com/102940764/194000644-ea36876c-d4eb-4766-89a0-c160e69df192.mp4

**Comment by jannahadlaon** [comment]
**Tested on:**
Android - MOCK_io.mosip.residentapp-0.4.0-rc5-newlogic_20221012_1559.apk
iOS - 0.4.0 (10.4)

MOSIP server: Mock

**Devices:**
Samsung Galaxy A23
iPhone 11

**Works as expected on ANDROID**
**FAILED ON iOS** @pmigueld @kyanthony @danicaerediano 

**STEPS**

1. Download UIN 
2. Click on the profile and wait for the green check beside **Valid** to appear
3. Click the kebab menu on the upper right corner
4. Select `Lock`
5. App should request OTP 
6. Enter correct OTP
7. Success toast should display
8. Click the kebab menu on the upper right corner again
9. Select `Unlock`
10. App should request OTP 
11.  Enter correct OTP
12. Success toast should display

**Additional comments/ notes**
- Lock/Unlock Feature does not work if the UIN is not Valid (Status has green check beside Valid in ID profile)
- Improve the success toast message to: **`ID successfully locked`**  -> @danicaerediano 
Screenshot_20221013-142635_MOSIP Resident App

**Comment by jannahadlaon** [comment]
Tested on:
Android - [MOCK_io.mosip.residentapp-0.4.0-rc6-newlogic_20221014_1757.apk](https://github.com/idpass/inji/releases/download/0.4.0-rc6/MOCK_io.mosip.residentapp-0.4.0-rc6-newlogic_20221014_1757.apk)
iOS - 0.4.0 (11.1)

MOSIP server: Mock

Devices:
Samsung Galaxy A23
iPhone 11

**WORKS AS EXPECTED for both android and iOS**
@pmigueld @danicaerediano  @kyanthony

**Pull request merged**